src/Main.java

```diff
@@ -1,33 +1,38 @@
 public class Main {
     public static void main(String[] args) {
 
-        System.out.println("Задание 1, 2");
+        System.out.println("Задание 1");
         int clientOS = 1;
-        int yearOfIssue = 2015;
-        int myYearOfIssue = 2014;
         if (clientOS == 1){
             System.out.println("Установите версию приложения для Android по ссылке.");
-            if (myYearOfIssue <= yearOfIssue){
-                System.out.println("Установите облегченную версию приложения для Android по ссылке.");
-            }
+
         }else if (clientOS == 0){
             System.out.println("Установите версию приложения для iOS по ссылке");
-            if (myYearOfIssue <= yearOfIssue){
-                System.out.println("Установите облегченную версию приложения для iOS по ссылке.");
-            }
 
             }
+        System.out.println("Задание 2"); // Поначалу думал, что нужно объеденить задание 1 и 2 =)
+        int yearOfIssue = 2015;
+        int myYearOfIssue = 2014;
+        if (clientOS == 1 && myYearOfIssue <= yearOfIssue) {
+            System.out.println("Установите облегченную версию приложения для Android по ссылке.");
+
+        }else if (clientOS == 1 && myYearOfIssue > yearOfIssue) {
+            System.out.println("Установите версию приложения для Android по ссылке.");
+
+        }else if (clientOS == 0 && myYearOfIssue <= yearOfIssue) {
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке.");
+
+        }else if (clientOS == 0 && myYearOfIssue > yearOfIssue) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        }
         System.out.println("Задание 3");
-        int year = 2021;
-        if (year % 4 == 0 || year % 400 == 0){
-            System.out.println(year + " год является високосным");
 
-        }else if (year % 100 == 0){
-            System.out.println(year + " год не является високосным");
+        int year = 1700;
+        if ((year % 4 == 0 && year % 100 != 0 && year > 1584) || (year % 400 == 0 && year > 1584)){
```

Comment:
```suggestion
        if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0 && year > 1584){
```
вот так сократить можно